File: main/src/main/java/net/fwitz/math/main/binary/escape/MandelcubedPlot.java
package net.fwitz.math.main.binary.escape;

public class MandelcubedPlot extends MandelbrotPlot {
    public static void main(String[] args) {
        new MandelcubedPlot().render();
    }

    public MandelcubedPlot() {
        super(3);
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/escape/MandelcubedPlot.kt
package net.fwitz.math.main.binary.escape

object MandelcubedPlot : MandelbrotPlot(
    name = "Mandelcubed (Escape Time)",
    power = 3.0
) {
    @JvmStatic
    fun main(args: Array<String>) = MandelcubedPlot.render()
}